### PR TITLE
Updated UI for list view for Resources and Shifting

### LINKED ISSUE #8732: Update UI for list view in shifting and resource boards
Current UI has a lot of whitespace

### PR DESCRIPTION
**Pull request by shivankacker**
## Proposed Changes

- Fixes #8732
- Reuses the same block UI for list view and board view
- Renamed files to match the same schema
- Cleaned up duplicate models

Screenshot 2024-10-27 at 1 09 17 AM

@ohcnetwork/care-fe-code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.ohc.network).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

**Comment by bodhish** [comment]
@shivankacker Can you check these PR's and get them reviewed and merged? I am seeing them have a lot of overlap with this PR. 

- Redesign UI for List View in Shifting #8882 [We are switching the list view to a table]
- Fixed Tabs View and added Strong Types #8899 (A minor PR adding fixed width for the search component)

**Comment by shivankacker** [comment]
@bodhish Didn't notice the Table UI PR before. The other PR will not interfere much. This PR adds a lot of changes for the structure of files and models. We can merge either of the PRs first but I think the Table UI one has some changes to do and has merge conflicts to resolve. We can merge this so the new file structure and models get reflected.

File: src/components/Shifting/ShiftingBoard.tsx [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+            ]}
+            onTabChange={(tab) =>
+              setBoardFilter(tab ? completedBoards : activeBoards)
+            }
+            currentTab={boardFilter[0].text !== activeBoards[0].text ? 1 : 0}
+          />
+
+          <div className="flex w-full flex-col gap-2 lg:mr-4 lg:w-fit lg:flex-row lg:gap-4">
+            <ButtonV2
+              className="py-[11px]"
```
Comment (rithviknishad) [inline src/components/Shifting/ShiftingBoard.tsx:115]: rem values preferred over arbitrary pixel values

File: src/components/Resource/ResourceList.tsx [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+              value={qParams.title}
+              onChange={(e) => updateQuery({ [e.name]: e.value })}
+              placeholder={t("search_resource")}
+            />
+          </div>
+          <div className="w-32">
+            {/* dummy div to align space as per board view */}
+          </div>
+          <div className="flex w-full flex-col gap-2 lg:w-fit lg:flex-row lg:gap-4">
+            <ButtonV2 className="py-[11px]" onClick={onBoardViewBtnClick}>
```
Comment (rithviknishad) [inline src/components/Resource/ResourceList.tsx:89]: rem over px

File: src/components/Shifting/ShiftingList.tsx [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+              onChange={(e) => updateQuery({ [e.name]: e.value })}
+              placeholder={t("search_patient")}
+            />
+          </div>
+          <div className="w-32">
+            {/* dummy div to align space as per board view */}
+          </div>
+          <div className="flex w-full flex-col gap-2 lg:w-fit lg:flex-row lg:gap-4">
+            <ButtonV2
+              className="py-[11px]"
```
Comment (rithviknishad) [inline src/components/Shifting/ShiftingList.tsx:130]: rem over px

**Review by rithviknishad — APPROVED** [review]
lgtm; just minor things

File: src/components/Shifting/ShiftingBoard.tsx (hunk 1 above)
Comment (shivankacker) [reply src/components/Shifting/ShiftingBoard.tsx:115]: Seems like they were added for the buttons on the top right to have the same height. No sub of 11px in TW, so letting it be : (

**Comment by nihal467** [comment]
LGTM

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**